presets/jscs.json

```diff
@@ -0,0 +1,14 @@
+{
+    "preset": "google",
```

Comment:
Good question, but I don't believe so. This is the definition of the jscs preset (which is an extension of the google preset).